website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'typescript-eslint-parser';
+
+const defaultOptions = {
+  range: true,
+  loc: false,
+  tokens: false,
+  comment: false,
+  tolerant: true,
+  useJSXTextNode: false,
+
+  ecmaFeatures: {
+    jsx: true,
+  },
+};
+
+const parserSettingsConfiguration = {
+  fields: [
+    'range',
+    'loc',
+    'tokens',
+    'comment',
+    'tolerant',
+    'useJSXTextNode',
+    {
+      key: 'ecmaFeatures',
+      title: 'ecmaFeatures',
+      fields: Object.keys(defaultOptions.ecmaFeatures),
```

Comment:
One caveat I noticed is that if you have a non-nested option after the nested one(s), it will also appear to be nested. Probably just a simple CSS fix needed, though.